From d4597491828bf4bee9e6520404c5dbb947979bde Mon Sep 17 00:00:00 2001
From: Park Ju Hyung <qkrwngud825@gmail.com>
Date: Wed, 19 Jun 2024 01:53:51 +0300
Subject: [PATCH] ssg: Set max available ratio to 25

Testing:
[ElectroPerf & resist15]
In testing we found out that there were significant improvements
in the sequential read and write speeds. Some screenshots of the tests are below:

Before: https://i.imgur.com/UBL74X2.jpg
After: https://i.imgur.com/CrkD5iE.jpg

Change-Id: Idd7f5c7df0a7fc1535555927923491ecb39bc6a9
[Tashar02: Apply patch on kernel]
Signed-off-by: Tashfin Shakeer Rhythm <tashfinshakeerrhythm@gmail.com>
---
 block/ssg-cgroup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/ssg-cgroup.c b/block/ssg-cgroup.c
index 1b36c95d2e74..cea830529053 100755
--- a/block/ssg-cgroup.c
+++ b/block/ssg-cgroup.c
@@ -50,7 +50,7 @@ static void ssg_blkcg_cpd_init(struct blkcg_policy_data *cpd)
 	if (IS_ERR_OR_NULL(ssg_blkcg))
 		return;
 
-	ssg_blkcg->max_available_ratio = 100;
+	ssg_blkcg->max_available_ratio = 25;
 }
 
 static void ssg_blkcg_cpd_free(struct blkcg_policy_data *cpd)
